File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
import java.io.IOException;
import java.sql.Array;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int yaksu = GCD(a,b);
        int basu = a*b/yaksu;

        System.out.println(yaksu);
        System.out.println(basu);

    }
    static int GCD(int a, int b){
        if(b==0){
            return a;
        }
        return GCD(b,a%b);
    }






}

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
import sys
input = sys.stdin.readline


def GCD(a, b):
    while b:
        a, b = b, a%b
    return a


cmd = list(map(int, input().split()))


yaksu = GCD(cmd[0],cmd[1])
basu = cmd[0]*cmd[1]/yaksu

print(int(yaksu), int(basu))